templates/articles/_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="comment-{{ comment.pk }}">
+<div class="card">
+    <div class="card-block">
+        <p class="card-text">
+            {{ comment.body }}
+        </p>
+    </div>
+    <div class="card-footer">
+        <a href="" class="comment-author">
+            <img
+                    src="{{ comment.author.img }}"
+                    class="comment-author-img"
+            />
+        </a>
+        &nbsp;
+        <a href="" class="comment-author">{{ comment.author }}</a>
+        <span class="date-posted">{{ comment.created_at }}</span>
+
+        {% if user.pk == comment.author.pk %}
```

Comment:
I think the `author` is actually a `Profile`. So you may need to compare `user.profile == comment.author`?

Reply:
Yea it is, odd that it worked my backwards way... corrected 👍🏻